bluebrain/documentation/setup_personal.md

```diff
@@ -2,22 +2,46 @@
 
 ## Building software on OS X
 
-Install software on OS X, using Homebrew for binary packages.
+To install end-user software on OS X, please defer to `brew`.
 
-On OS X the build process is very similar to Ubuntu. To avoid building
-the whole stack from source we can likewise use another package manager to provide precompiled binaries.
-To that end we have successfully used Homebrew. We also provide a skeleton 
-`packages.yaml` that you should review and adapt to your needs.
+Before starting, please install `brew` and make sure that there is a
+working Python on your machine, preferably from XCode or another *stable*
+source.
```

Comment:
Just about this - I prefer this as well but in the past have seen (undocumented, minor) issues when using system python. Your didn't see anything major on your Apple M1 test?

Reply:
I used that Python to bootstrap a Spack one 🤷

Reply:
FWIW, building Python itself was faster than building CMake…

Reply:
> I used that Python to bootstrap a Spack one

Ah ok 👍